packages/google_maps_flutter/lib/src/marker_updates.dart

```diff
@@ -0,0 +1,186 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of google_maps_flutter;
+
+/// Types of [Marker] updates.
+enum MarkerUpdateEventType {
+  add,
+  update,
+  remove,
+}
+
+Map<MarkerId, Marker> _toMap(Set<Marker> markers) {
```

Comment:
See suggestion on _createMarkerIdMarkerMap, also this seems to be redundant with the logic there?